com.unity.multiplayer.mlapi/Runtime/Core/NetworkedObject.cs

```diff
@@ -493,9 +493,19 @@ internal void InvokeBehaviourOnGainedOwnership()
 
         internal void ResetNetworkedStartInvoked()
         {
-            for (int i = 0; i < childNetworkedBehaviours.Count; i++)
+            try
+            {
+                if(childNetworkedBehaviours != null)
+                {
+                    for (int i = 0; i < childNetworkedBehaviours.Count; i++)
+                    {
+                        childNetworkedBehaviours[i].networkedStartInvoked = false;
+                    }
+                }
+            }
+            catch(Exception ex)
             {
-                childNetworkedBehaviours[i].networkedStartInvoked = false;
+                Debug.LogError(ex);
```

Comment:
did you forget to remove try-catch block here?

Reply:
Good catch, that is removed.